app/scripts/metamask-controller.js

```diff
@@ -3379,11 +3376,10 @@ export default class MetamaskController extends EventEmitter {
   getState() {
     const { vault } = this.keyringController.state;
     const isInitialized = Boolean(vault);
-    const flatState = this.memStore.getFlatState();
 
     return {
       isInitialized,
-      ...sanitizeUIState(flatState),
+      ...sanitizeUIState(this.memStore.getState()),
```

Comment:
Minor, but still nice to have a dedicated variable for this.

Reply:
Applied here: https://github.com/MetaMask/metamask-extension/pull/28723/commits/d83796f39a18e0e3b09a29c0f96d4278f48ce6fc